styles.css

```
@@ -0,0 +1,339 @@
+}
+
+h1 { font-size:42px;
+    line-height:1.3;
+    padding-bottom:20px;}
+
+h2 { font-size:42px;
+    line-height:1.3;
+    padding-bottom:20px;
+    color:#fefefe;
+}
+
+
+
+h3 {
+    font-size: 28px;
+    line-height: 1.5;
+    color: var(--blue);
+    padding: 12px 0;
+}
+
+
+
+h5 {
+    font-size: 23px;
+    line-height: 1.5;
+    color: var(--purple);
+    padding: 10px 0 20px 0;
+}
+
+
+
+p {
+    font-family: Roboto, Arial, serif;
+    font-size:21px;
+    line-height:1.5;
+    font-weight:500;
+    color: var(--text-black);
+}
+
+
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+
+
+
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-content {
+    width:100%;
+    display:flex;
+    flex-direction:row;
+    justify-content:center;
+}
+
+.inner-content {
+    display:flex;
+    flex-direction:column;
+    justify-content:center;
+    align-items:center;
+    flex-basis:1400px;
+    flex-shrink:1;
+    flex-grow:0;
+    margin: 0 20px;
+
+}
+
+.intro {
+    text-align:center;
+    max-width:800px;
+}
+
+.content-area {
+    padding:160px 0 160px 0;
+
+}
+
+
+/* --------------- 3.2 Other --------------------------- */
+
+button {
+    font-size: 14px;
+    font-weight: 300;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    color: white;
+    background-color: var(--pink);
+    border: none;
+    border-radius: 60px;
+    box-shadow: 0 4px 7px 4px rgba(0, 0, 0, .2);
+    padding: 20px 40px;
+    margin: 28px 0;
+}
+
+button:hover {
+    background-color:#d03672;
+    box-shadow: 0 2px 4px 2px rgba(0, 0, 0, .3);
+    cursor: pointer;
+}
+
+
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+header { background-color:var(--blue);
+    background: linear-gradient(130deg, var(--blue) 0%, var(--purple)  100%);
+    color: white;
+    position:relative;
+    z-index:1;
+}
+
+header::after {
+    content: '';
+    width: 78px;
+    height: 100px;
+    position:absolute;
+    bottom: 0;
+    left: calc(50% - 39px);
+    background-image: url(assets/squiggle_line_stripe.svg);
+}
+
+
+
+
+nav {  display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+    margin:30px;
+}
+
+nav ul  { list-style:none;
+    flex-basis:350px;
+    flex-shrink:1;
+    flex-grow:0;
+    display:flex;
+    flex-direction:row;
+    justify-content:space-between;
+
+}
+
+nav a { color:white; text-decoration:none; }
+
+/* --------------- 4.2 Case studies ------------------- */
+
+#casestudies {
+
+    background-color: var(--text-grey);
+
+
+}
+
+.case {
+    display:flex;
+    flex-direction:row;
+    justify-content:space-between;
+    align-items:center;
+    margin:0 0 100px 0;
+
+}
+
+.case:nth-of-type(even) {
+    flex-direction:row-reverse;
+}
+
+.case-info {
+    flex-basis:480px;
+    flex-grow:0;
+    flex-shrink:0;
+    padding:40px; 40px 40px 40px;
```

Je kan hier een shorthand voor gebruiken
```suggestion
    padding:40px;
```